src/libraries/System.Private.CoreLib/src/Internal/Runtime/CompilerServices/Unsafe.cs

```diff
@@ -13,15 +12,10 @@
 // In CoreRT, see Internal.IL.Stubs.UnsafeIntrinsics for details.
 //
 
-namespace Internal.Runtime.CompilerServices
+namespace System.Runtime.CompilerServices
 {
-    //
-    // Subsetted clone of System.Runtime.CompilerServices.Unsafe for internal runtime use.
-    // Keep in sync with https://github.com/dotnet/runtime/tree/main/src/libraries/System.Runtime.CompilerServices.Unsafe.
-    //
-
     /// <summary>
-    /// For internal use only. Contains generic, low-level functionality for manipulating pointers.
+    /// Contains generic, low-level functionality for manipulating pointers.
     /// </summary>
     [CLSCompliant(false)]
```

Comment:
The attributes between impl and ref should match as much as possible. If the ref is marked per-method, the impl should be marked like that as well.

Reply:
Done.